documentation/modules/ref-address-space-example-exporting-endpoints.adoc

```diff
@@ -28,7 +28,7 @@ spec:
       name: my-config <2>
 ----
 <1> (Required) The export type. The resulting `ConfigMap` contains the values in the format
```

Comment:
Suggested change:

(Required) The type of export, either `ConfigMap`, `Secret`, or `Service`.